Client/Frontend/Widgets/Toolbar.swift

```diff
@@ -1,93 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import Foundation
-import SnapKit
-
-class Toolbar: UIView {
```

Comment:
Toolbar was used to show the StackView buttons and the TabToolbar. Both have so little to do with each other that most people think that `Toolbar` is only used in TabToolbar. 

Decided to get rid of Toolbar completely and have Snackbar use stackviews instead.